content/index.md

```diff
@@ -0,0 +1,172 @@
+
+## Interactive Commands
+
+**All commands must be available in non-interactive mode** to enable the tool to be used in automation.
+
+For users who are not familiar with the tool, the tool offer interactive prompts for the parameters that a command supports. So for example using `number:search` without any further commands causes the tool to prompt the user for a country code (probably with a link to codes list, this might not be a great example!) and then offer the optional parameters with the option to accept the defaults.
+
+## Output Format
+
+Some commands return data, some simply perform an action.
+
+### Action Commands
+
+Action commands return any output that is needed, even if it is just "OK". They also return an appropriate exit status.
+
+All non-data-returning commands support a `--quiet` switch to suppress output.
+
+### Data Comands
+
+Any command that returns data should do so in the following ways.
+
+**Single value**
+
+A command that returns a single value can simply output that value.
+
+**Multiple values**
+
+Even for a single row, this data should be returned in columnar format, with headings. Example:
+
+```
+msisdn       | country | type       | features  | voiceCallbackType | voiceCallbackValue                   | moHttpUrl                                        | voiceStatusCallbackUrl
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+442038795229 | GB      | landline   | VOICE     | app               | 3050a697-f298-4601-8476-7805eafd33fc | undefined                                        | undefined             
+442038931433 | GB      | landline   | VOICE     | app               | 048f0fb1-39a8-40c4-1149-6a2260771fa1 | undefined                                        | undefined             
+442038931435 | GB      | landline   | VOICE     | app               | 048f0fb1-39a8-40c4-9d11-6a2260771fa1 | undefined                                        | undefined             
+447418340330 | GB      | mobile-lvn | VOICE,SMS | undefined         | undefined                            | undefined                                        | undefined             
+447418343340 | GB      | mobile-lvn | VOICE,SMS | undefined         | undefined                            | https://example.ngrok.io/webhooks/inbound-sms | undefined             
+447418343838 | GB      | mobile-lvn | VOICE,SMS | app               | a7fb2d02-92d6-43b4-ae58-6011eae58bbc | https://example.ngrok.io/incoming/sms         | undefined             
+```
+
+This standard approach lends itself well to use with common commands such as `cut` and `tail` that are in common use by developers.
+
+**JSON Format**
+
+For commands that return multiple values, adding a `--format-json` switch will return a pretty-printed JSON response instead of the standard data output.
+
+**Pagination**
```

Comment:
Should we add something about the use of terminal pager for large responses displayed in the terminal?

Reply:
I didn't add this, users might want to do something else with the output